alws/models.py

```diff
@@ -1446,37 +1448,37 @@ class NewErrataRecord(Base):
     definition_class: Mapped[str] = mapped_column(
         sqlalchemy.Text, nullable=False
     )
-    affected_cpe: Mapped[Dict[str, Any]] = mapped_column(
+    affected_cpe: Mapped[List[Dict[str, Any]]] = mapped_column(
```

Comment:
Did you check the changes related to oval cacher locally?

Reply:
Payload in oval_cacher.py has `affected_cpe: advisory.affected_cpe_list` and `affected_cpe_list`is marked as `List[str]` in schema.py.
Also other List fields in `ErrataRecord` and `NewErrataRecord` seems to be fine.
Not sure about `criteria` (`criteria = xml_oval_def["criteria"]` in payload) but it also seems to be a list